files/en-us/related/epub/structure/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'EPUB Structure'
+slug: Related/EPUB/Structure
+tags:
+  - EPUB
+  - Ebook
+  - Ebooks
+  - Guide
+  - Reference
+---
+{{EPUBRef}}
+
+An unzipped EPUB file is a folder containing two primary directories and a `mimetype` file. One of
```

Comment:
To avoid a gerund, how about "a folder that contains"?

Reply:
```suggestion
An unzipped EPUB file is a folder that contains two primary directories and a `mimetype` file. One of
```